### PR TITLE
Ft new york times

### PR DESCRIPTION
**Pull request by Eder-PG**
Features branch for the New York Times project

Third project ready for submission
No HTML or CSS-Lint errors left

Added full content for:
Index.html
styles.css
README.md

Added files
LICENCE file
20+ images in the assets folder

EDIT: formatted better the request text and applied required changes to meet review requirements

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,568 @@
...
+                        <h6> These Stand-Ups Are in an Escapist Moon, Much Like Their Audience.</h6>
+                        <span class="suggested-date">Dec, 13</span>
+                    </div>
+                </div>
+            </div>
+            <div class="most-popular">
+                <div class="suggestions-title">
+                    <!--<svg class="suggestions-title-line">
+                    <line x1="0" y1="0" x2="50%" y2="0" />
+                </svg>-->
```
Comment (pelzolga123) [inline index.html]: Commented code should be removed.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,568 @@
...
+                        <p class="suggested-credits">NASA:Goddard</p>
+                        <h6>NASA's Parker Solar Probe Is Unlocking the Sun's Mysteries</h6>
+                        <span class="suggested-date">Dec. 4</span>
+                    </div>
+                </div>
+                <div class="editor-picks">
+                    <div class="suggestions-title">
+                        <!--<svg class="suggestions-title-line">
+                            <line x1="0" y1="0" x2="50%" y2="0" />
+                        </svg>-->
```
Comment (pelzolga123) [inline index.html]: Commented code should be removed.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,568 @@
...
+            </div>
+        </article>
+        <div class="suggestions">
+            <div class="extra-articles">
+                <div class="space-cosmos">
+                    <div class="suggestions-title">
+                        <hr>
+                        <!--<svg class="suggestions-title-line">
+                            <line x1="0" y1="0" x2="50%" y2="0" />
+                        </svg>-->
```
Comment (pelzolga123) [inline index.html]: Commented code should be removed.

**Review by pelzolga123 — CHANGES_REQUESTED** [review]
Logo and bottom part should have links. Readme file needs some correction. Pull request should be more descriptive.

**Comment by pelzolga123** [comment]
## Hi Team!  
#### Your project looks :+1: amazing!

Here are some things, that require changes:

* [ ] Pull Request has meaningful title and a short description. You have some text in your pull request description section, but it is not the right one. Name should be changes as well to more descriptive. Correct pull request description should have list of things you implemented in this project. 
For example:

* Created fixed header and footer.
* Added embedded images and video.

Look at [this link](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243642/) from pathwright for detailed description.

* [ ] Readme file has 'HTML Forms' header. This header is from previous project, please change it.  

* [ ] In Navbar NY Times logo should be a link. [Link to original website](https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html?_r=0)

* [ ] Bottom part should contain links to other articles on the leftmost side. [Link to original website](https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html?_r=0)

* [ ] Bottom part should contain links to other articles on the rightmost side. [Link to original website](https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html?_r=0)

* [ ] Footer should have a fixed position (it sticks to the bottom part of the screen):
image

### Please kindly fix the above issues and send another code review request.

If you have any questions or need help, you can message me on slack (@ Olga Pelts). *Happy coding*

**Comment by Eder-PG** [comment]
Thanks! I added all missing features and fixed oversights pointed by the review
Code is still free of HTML and CSS errors

**Comment by izaiasneto4** [comment]
# Hi folks :wave: 
Great work implementing the changes. Looks like you have everything. It's a yes for me :+1: 

Your project looks really good by the way :wink: 

You can merge the project and submit the project completion form :+1: :100:

**Comment by kbjude** [comment]
Thank you Izaias,
Thats very encouraging.

**Pull request merged**